components/pkg-export-helm/plan.sh

```diff
@@ -0,0 +1,80 @@
+pkg_name=hab-pkg-export-helm
```

Comment:
Don't forget to add this package to `.bldr.toml` and to [this line](https://github.com/habitat-sh/habitat/blob/b4fa6df76d4c4f8f7f87e1898563eda52db16d0a/.travis.yml#L256) of `.travis.yml` to ensure that things are properly picked up in the builds

Reply:
Sure.